common/djangoapps/third_party_auth/models.py

```diff
@@ -347,6 +347,9 @@ class SAMLProviderConfig(ProviderConfig):
     attr_email = models.CharField(
         max_length=128, blank=True, verbose_name="Email Attribute",
         help_text="URN of SAML attribute containing the user's email address[es]. Leave blank for default.")
+    include_in_saml_pull = models.BooleanField(
```

Comment:
How about just expanding the help text to say "If the metadata is at a publicly accessible URL, this should always be enabled so that the metadata stays up to date. Disable if there is no publicly accessible URL for the metadata."